src/js/letters/actions/letters.js

```diff
@@ -130,29 +130,24 @@ export function getLetterPdf(letterType, letterName, letterOptions) {
     };
   }
 
-  // We handle IE10 separately but assume all other vets.gov-supported
-  // browsers have blob URL support.
-  // TODO: possibly want to explicitly check for blob URL support with something like
-  // const blobSupported = !!(/^blob:/.exec(downloadUrl));
-  const ie10 = !!window.navigator.msSaveOrOpenBlob;
-  const save = document.createElement('a');
-  let downloadWindow;
-  const downloadSupported = typeof save.download !== 'undefined';
-  if (!downloadSupported) {
-    // Instead of giving the file a readable name and downloading
-    // it directly, open it in a new window with an ugly hash URL
-    downloadWindow = window.open();
-  }
-  let downloadUrl;
   return (dispatch) => {
     dispatch({ type: GET_LETTER_PDF_DOWNLOADING, data: letterType });
     apiRequest(
       `/v0/letters/${letterType}`,
       settings,
       response => {
+        let downloadUrl;
         response.blob().then(blob => {
+          // We handle IE10 separately but assume all other vets.gov-supported
+          // browsers have blob URL support.
+          // TODO: possibly want to explicitly check for blob URL support with something like
+          // const blobSupported = !!(/^blob:/.exec(downloadUrl));
+          const ie10 = !!window.navigator.msSaveOrOpenBlob;
+          const save = document.createElement('a');
+          const downloadSupported = typeof save.download !== 'undefined';
+
           if (ie10) {
-            window.navigator.msSaveOrOpenBlob(blob, letterName);
+            window.navigator.msSaveOrOpenBlob(blob, `${letterName}.pdf`);
```

Comment:
:crossed_fingers: Hopefully this works; need to test it. The file previously downloaded just fine, but without the extension. Manually adding the extension made it work like a charm, so hopefully this does the trick.